network/gossip/libp2p/libp2pNode_test.go

```diff
@@ -48,19 +46,18 @@ func TestLibP2PNodesTestSuite(t *testing.T) {
 }
 
 // SetupTests initiates the test setups prior to each test
-func (l *LibP2PNodeTestSuite) SetupTest() {
-	l.logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr}).With().Caller().Logger()
-	l.ctx, l.cancel = context.WithCancel(context.Background())
-	golog.SetAllLoggers(golog.LevelDebug)
+func (suite *LibP2PNodeTestSuite) SetupTest() {
+	suite.logger = zerolog.New(os.Stderr).Level(zerolog.DebugLevel)
+	suite.ctx, suite.cancel = context.WithCancel(context.Background())
```

Comment:
This is a refactoring to several test suites in the network package, we set all their log level to error to avoid having unnecessary debug logs of the internals.

Reply:
agree 100%